inference-engine/tests/functional/inference_engine/lp_transformations/add_transformation.cpp

```diff
@@ -152,6 +152,8 @@ TEST_P(AddTransformation, CompareFunctions) {
     actualFunction->validate_nodes_and_infer_types();
     auto res = compare_functions(referenceFunction, actualFunction, true, true, false);
     ASSERT_TRUE(res.first) << res.second;
+
+    ASSERT_TRUE(LayerTransformation::allNamesAreUnique(actualFunction)) << "Not all names are unique";
```

Comment:
minor, not for update right now: ideally the validation has to be added in base type

Reply:
my bad, you are right: not an issue